oneanddone/tests/functional/tests/test_task_assignment.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subjectfrom django import forms to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestAvailableTasks():
+    @pytest.mark.nondestructive
+    def test_create_tasks(self, base_url, selenium, task, random_title):
+
+        admin_user = {'email': "varna.suresh@outlook.com",
+                      'password': "varna123",
+                      'name': "Varna",
+                      'username': "Varna",
+                      'url': 'http://www.mozilla.org/'
+        }
```

Comment:
If you're unable to log in or access certain areas of the site it may be due to some changes we've made due to these credentials being shared. Let me or someone in #mozwebqa on IRC know if you're seeing issues and we can work to resolve them.